HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -2131,6 +2131,42 @@ def self.apply_hot_water_distribution(hpxml_bldg, cfa, ncfl, has_uncond_bsmnt)
         hot_water_distribution.shared_recirculation_pump_power_isdefaulted = true
       end
     end
+
+    if hot_water_distribution.system_type == HPXML::DHWDistTypeRecirc || hot_water_distribution.has_shared_recirculation
+      schedules_file_includes_recirculation_pump = (schedules_file.nil? ? false : schedules_file.includes_col_name(SchedulesFile::ColumnHotWaterRecirculationPump))
+      if [HPXML::DHWRecircControlTypeNone, HPXML::DHWRecircControlTypeTimer].include?(hot_water_distribution.recirculation_control_type) || [HPXML::DHWRecircControlTypeNone, HPXML::DHWRecircControlTypeTimer].include?(hot_water_distribution.shared_recirculation_control_type)
```

Comment:
Simplify the `if` statements by using...

```suggestion
      recirc_control_type = hot_water_distribution.has_shared_recirculation ? hot_water_distribution.shared_recirculation_control_type : hot_water_distribution.recirculation_control_type
      if [HPXML::DHWRecircControlTypeNone, HPXML::DHWRecircControlTypeTimer].include?(recirc_control_type)
```